sumo-api/sumo/metrics/evaluator.py

```diff
@@ -577,7 +619,153 @@ def _print_sim_cache(self, sim_cache):
                 corr = sim_cache[det_id][gt_id]
                 print("Det: {} | GT: {} | sim: {} | score: {}".format(det_id, gt_id, corr.similarity, corr.det_score))
 
-                
+
+    def _perceptual_score_1(self, matched, missed, extra, normalized=True):
+        """
+        Compute perceptual score for a given data association at a
+        single similarity threshold. 
+
+        Inputs:
+        matched (dict of Corrs) - dictionary of correspondences (from
+          data association) between submitted elements and
+          ground-truth elements. 
+        missed (set of object_ids) - ids of non-matched ground-truth instances
+        extra (set of object_ids) - ids of non-matched detected instances
+        normalized (Bool) - if True, normalize the score (from 0-1),
+          else return the raw score
+
+        Return:
+        perceptual_score (float) - larger score is better
+        """
+        perceptual_score = 0
+        max_score = 0
```

Comment:
max_score doesn't seem to be used anywhere

Reply:
It is used at the end of the routine for normalization.